### PR TITLE
Task/task 72 dashboard application redirect

### PR DESCRIPTION
**Pull request by kulanthairavenantech**

### CODE REVIEW

File: front/src/app/applications/[id]/page.tsx [hunk 1]
```diff
@@ -1,103 +1,109 @@
...
-  }));
+    useEffect(() => {
+        if (typeof window !== "undefined") {
+            const application = localStorage.getItem("application-data");
+            if (application) {
+                setApplicationData(JSON.parse(application));
+            }
+        }
+    }, []);
+    console.log(applicationData);
```
Comment (vsaravanakumar) [inline front/src/app/applications/[id]/page.tsx:46]: Remove console.log

File: front/src/components/CustomCard/CustomCard.tsx [hunk 2]
```diff
@@ -1,65 +1,103 @@
...
+    };
+
+    const handleMouseLeave = () => {
+        setHovered(false);
+    };
+
+    const handleClick = async () => {
+        try {
+            if (role?.toLowerCase() === "client") {
+                console.log(role);
```
Comment (vsaravanakumar) [inline front/src/components/CustomCard/CustomCard.tsx:45]: Remove console.log

File: front/src/components/Login/LoginForm.tsx [hunk 3]
```diff
@@ -1,237 +1,348 @@
...
-            }}
-            variant="outlined"
-            error={Boolean(errors.username)}
-            helperText={
-              errors.username ? errors.username.message?.toString() : null
+    useEffect(() => {
+        if (key && clientId) {
+            goToApp();
+        }
+    }, [key, clientId]);
```
Comment (vsaravanakumar) [inline front/src/components/Login/LoginForm.tsx:86]: Solve Eslint error

File: front/src/components/Login/LoginForm.tsx [hunk 4]
```diff
@@ -1,237 +1,348 @@
...
+                clientId!
+            );
+            if (response.statusCode == 200 && response.applicationId) {
+                const result = await UserApi.quickSignIn({
+                    userId: session.user.id,
+                    applicationId: response.applicationId,
+                });
+                const { apiToken, callBackUrl } = result;
+                if (apiToken && callBackUrl) {
+                    window.location.href = `${callBackUrl}?code=${apiToken}`;
```
[the remark this answers is not in the document]
Comment (kulanthairavenantech) [reply front/src/components/Login/LoginForm.tsx]: passed client details with token to external app

File: front/src/components/Login/LoginForm.tsx (hunk 3 above)
Comment (kulanthairavenantech) [reply front/src/components/Login/LoginForm.tsx]: Solved eslint error

File: front/src/components/CustomCard/CustomCard.tsx (hunk 2 above)
Comment (kulanthairavenantech) [reply front/src/components/CustomCard/CustomCard.tsx]: Removed unwanted console.log

File: front/src/app/applications/[id]/page.tsx (hunk 1 above)
Comment (kulanthairavenantech) [reply front/src/app/applications/[id]/page.tsx]: Removed unwanted console.log

**Pull request merged**